### PR TITLE
Patch Unit Tests

### PR DESCRIPTION
**Pull request by theczorn**
- Patched chai unit tests
- ESLint fixes

### CODE REVIEW

File: tests/unit/src/commands/clash.spec.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ describe('mtg clash command', function () {
     describe('clash', function () {
         describe('when', function () {
             it('should', function () {
-                expect(true).to.be.truthy();
```
Comment (theczorn) [inline tests/unit/src/commands/clash.spec.js:32]: AFIACT, the ".truthy()" function does not exist within (Sinon-)Chai.

Standard Chai syntax is using the ".ok" prop to establish "truthiness"
http://chaijs.com/api/bdd/#ok

Comment (HerrPfister) [reply tests/unit/src/commands/clash.spec.js:32]: Interesting, I wonder when that changed? Good to know 👍

File: tests/unit/src/commands/clash.spec.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var mtgClash = require('../../../../src/commands/clash'),
-    mtg = requre('mtgsdk'),
+    mtg = require('mtgsdk'),
```
Comment (HerrPfister) [inline tests/unit/src/commands/clash.spec.js:4]: good catch. I'm confused as to why this didn't break when I ran the tests before ...

**Comment by HerrPfister** [comment]
Looks good +1

**Pull request merged**